pkg/common/hashmap/strhashmap.go

```diff
@@ -61,6 +61,9 @@ func (m *StrHashMap) AddGroup() {
 // never handle null
```

Comment:
Is this the right way to do insert value?    How about just do InserValue([][]byte) and caller just do InsertValue(enconding.EncodeFixed(v))?

Reply:
use Iterator.Insert